### PR TITLE
Swift5 upgrade

### PR DESCRIPTION
**Pull request by vexy**
Build settings updated to use Swift 5
AppDelegate function parameter rename accordingly
Minor linting screams prevention :)

### CODE REVIEW

File: Atom Attack/Extensions/CGFloat+Extensions.swift [hunk 1]
```diff
@@ -11,9 +11,11 @@ import CoreGraphics
 extension CGFloat {
 
     static func random() -> CGFloat {
-        return CGFloat(Float(arc4random()) / 0xFFFFFFFF)
+        let seed: Float = Float(0xFFFFFFFF)                //explicit Float cast removes compiler warnings on precision issues (remove Float cast to see original warning)
+        return CGFloat(Float(arc4random()) / seed)
```
Comment (VladislavJevremovic) [inline Atom Attack/Extensions/CGFloat+Extensions.swift]: Today I'd just use `return CGFloat.random(in: 0.0...1.0)` here 😉

Comment (vexy) [reply Atom Attack/Extensions/CGFloat+Extensions.swift]: Totally true. But in this way the code follows the overall (good old C) style seen in other methods :)) More true Swift updates to follow

Comment (VladislavJevremovic) [reply Atom Attack/Extensions/CGFloat+Extensions.swift]: I had recently planned to bring this one up to modern standards, but you always lack the time. Let's modernize this.

Comment (vexy) [reply Atom Attack/Extensions/CGFloat+Extensions.swift]: I'm totally down 😄 
My plan was not only to refresh it to Swift5, but also to refactor some pieces of code so the game overall can be more extensible. Say, adding new "enemies" (read atoms), different colors, leveling system revamp, etc.. Nothing special, just for the sake of fun.

My fork contains entire development, drop an 👀  sometimes if you find something useful.

Happy to hear other suggestions on this interesting and funny piece of Swift knowledge :))

Cheers 🍺

**Pull request merged**